src/analysis/normed_space/operator_norm.lean

```diff
@@ -381,14 +382,89 @@ lemma op_norm_smul_le {𝕜' : Type*} [normed_field 𝕜'] [normed_space 𝕜' F
   end))
 
 /-- Continuous linear maps themselves form a seminormed space with respect to
-    the operator norm. -/
-instance to_seminormed_add_comm_group : seminormed_add_comm_group (E →SL[σ₁₂] F) :=
+the operator norm. This is only a temporary definition because we want to replace the topology
+with `continuous_linear_map.topological_space` to avoid diamond issues.
+See Note [forgetful inheritance] -/
+private def tmp_seminormed_add_comm_group : seminormed_add_comm_group (E →SL[σ₁₂] F) :=
```

Comment:
I think the current rule in mathlib is: no private stuff ever. Because it breaks goto definition and sometimes documentation, and you never know what users might want to do.